map.js

```diff
@@ -86,14 +86,15 @@ function Panel(element) {
 	this._hide.title = lang.action_collapse;
 	this._hide.className = 'hide';
 	this._hide.addEventListener('click', this.toggleExpanded.bind(this));
-	
-	this._close = addElementWithText(this._element, 'a', '×');
+
+	this._content = document.createElement('div');
```

Comment:
I would propose to leave the content as it was (it should only contain elements passed in `show` method parameter, not some-content-and-hide-button) and create a new element, maybe `_container` (having `panel-container` class), that would wrap `_hide` and `_content`.